src/process_model_output.R

```diff
@@ -51,14 +52,14 @@ find_quantile_group <- function(value, breaks, labels) {
 
 percentile_categories <- c("very low", "low", "average", "high", "very high")
 values_categorized <- total_storage_data %>%
-  left_join(quantile_df, by = c("hruid", "DOY")) %>%
-  mutate(map_cat = find_value_category(total_storage_today, total_storage_quantiles, percentile_categories)) %>%
+  left_join(quantile_df, by = c("hruid")) %>%
+  mutate(map_cat = find_quantile_group(total_storage_today, total_storage_quantiles, percentile_categories)) %>%
```

Comment:
locally I see this issue when I run this code block starting at line 54
```
> values_categorized <- total_storage_data %>%
+   left_join(quantile_df, by = c("hruid")) %>%
+   mutate(map_cat = find_quantile_group(total_storage_today, total_storage_quantiles, percentile_categories)) %>%
+   select(hru_id_nat = hruid,
+          value = as.character(map_cat))
Error in sort.int(as.double(breaks)) : 
  (list) object cannot be coerced to type 'double'
In addition: Warning message:
 Error in sort.int(as.double(breaks)) : 
  (list) ```

Reply:
Looks like somewhere there is a data frame with a column that is a list of lists where a numeric is expected.  I would guess its the column that contained the HRU quantiles, I think that was a list, and probably needs to be broken out into separate columns for `find_quantile_group` to work.

Reply:
I've fixed this on my fork, fix will come with my PR